### ISSUE TITLE
How?

### ISSUE THREAD

**Issue by CreatedPNG** [Mondmarmelade/Kahboom#9]
Hello,

I'm new to programming and so on! Could someone say me how they got the awnsers, and
Python with HTML how, wasn't it JavaScript?
Thanks!

CreatedPNG

**Comment by StandartCoder** [comment]
Hello my name is StandartCoder,

I'm an Contributor of this Project! So first off welcome to programming, and good luck in your next projects!
Now we come to the main topic: How?
So its pretty simple how this works, we are running an Flask Server and this Flask Server waits until someone sends an request to `/`. (Ex: https://kahboom.tk/) After he receives one, he just sends the user an index.html file, with css, js linked!
So this Python Flask Server just handels requests!

Python - Handler
HTML, CSS, JS - Frontend

And now how we get the awnsers?
So basically, there is an rest api from Kahoot themselfs! So we just send an request with the quizID and get the responds and give it out with HTML!

I hope this is all understandble, and this is all!
Good, just reply if this was all! And I can close this issue!
Happy coding!

Kind regards
~StandartCoder (Contributor)

**Comment by CreatedPNG** [comment]
Thank you for this awnser!
I now understand how it works!

CheckedPNG
